test/integration/imagerepository_test.go

```diff
@@ -31,6 +31,14 @@ func TestImageRepositoryList(t *testing.T) {
 		t.Errorf("unexpected error: %v", err)
 	}
 
+	clusterAdminKubeClient, err := testutil.GetClusterAdminKubeClient(clusterAdminKubeConfig)
```

Comment:
Create an issue to fix these tests.  The tests should follow our expected flow of, "create a project, add an admin, make an editor (optional), run tests as the editor".  That flow will test to make sure all permissions are correct as well.  This one sort of side steps it.